### DISCUSSION TITLE
Timeout

### DISCUSSION THREAD

**Discussion by JonathanCR2021** [kimai/kimai#2674] [Questions & Answers]
Hello,
I have the Kimai 1.14.1. The problem that users complain to me is that the time that the system remains active is very short.
In other words, they activate a function, and when they go to create a new function, the system asks for the password again. This has them crazy because it is only a matter of a few minutes.
How do I eliminate or increase the closing time?

**Comment by kevinpapst** [comment]
Configure your PHP to have an appropriate Session lifetime (the default lifetime on new servers are 14 minutes).

You might be lucky by setting another lifetime inside Kimai: https://www.kimai.org/documentation/users.html#session-lifetime

But normally you should instruct your PHP to have an appropriate lifetime, because this should not be an application setting.

**Reply by JonathanCR2021** [reply]
Hello, I will try to make the change.
Thank you!
